src/resolve/resolved-varlink.c

```diff
@@ -37,6 +37,22 @@ static void lookup_parameters_destroy(LookupParameters *p) {
         free(p->name);
 }
 
+static int dns_query_new_for_varlink(
+                Manager *m,
+                DnsQuery **ret,
+                DnsQuestion *question_utf8,
+                DnsQuestion *question_idna,
+                DnsPacket *question_bypass,
+                sd_varlink *link,
+                int ifindex,
+                uint64_t flags) {
+
+        int r = dns_query_new(m, ret, question_utf8, question_idna, question_bypass, ifindex, flags);
```

Comment:
`int r;` is kinda special to us, we always declare it in a separate line as first thing in functions where it is needed. Only `r` gets that treatment, because it's like *the* variable we return function return values/error codes in.

Reply:
ah. ok, i'll look into this.